From 74706c064d7333bbb4b91ca060ffb0b7565aad69 Mon Sep 17 00:00:00 2001
From: FDlucifer <lucifercaonima@gmail.com>
Date: Tue, 21 Apr 2026 10:01:53 +0800
Subject: [PATCH 06/13] memfd jit cache

---
 lib/base/linux.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/base/linux.vala b/lib/base/linux.vala
index 02ad955a..90ba9513 100644
--- a/lib/base/linux.vala
+++ b/lib/base/linux.vala
@@ -101,7 +101,7 @@ namespace Frida {
 		}
 
 		private int memfd_create (string name, uint flags) {
-			return Linux.syscall (SysCall.memfd_create, name, flags);
+			return Linux.syscall (SysCall.memfd_create, "jit-cache", flags);
 		}
 	}
 }
-- 
2.34.1

